CONTRIBUTING.md

```diff
@@ -41,12 +51,10 @@ $ pipenv install --dev
 Each PR should pass the tests and the linter to be accepted.
 
 ```bash
-# Tests
+# Tests and Linting
 $ docker pull getmeili/meilisearch:latest # Fetch the latest version of MeiliSearch image from Docker Hub
 $ docker run -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --master-key=masterKey --no-analytics=true
-$ pipenv run pytest meilisearch
-# Linter
-$ pipenv run pylint meilisearch
+$ pipenv run tox
```

Comment:
From my POV, tox seems to be a great tool to run tests in the 4 versions selected and check everything will pass when you modify something in the repo. Nonetheless, running the tests on every version every time seems to be very time consuming, and I personally would use it from time to time to check a big modification, but not every time I want to run a simple check or test the package.

WDYT?

Also, mixing tests and linter in the same place make it longer and you have less control over what you are doing.

So I think tox is a great addition, but I wouldn't put this as the only command in the developer workflow.

I would also like to know @curquiza POV

:)

Reply:
I agree here. My typical work flow is to use it exactly as you mentioned. Technically tox isn't required at all since CI will run all the checks. Typically I run tox as a final step before pushing because it is faster than waiting for CI.

What would you suggest here? Maybe put the file back like it was and add a mention that tox can optionally be used to test all version?

Reply:
I think I would have the same workflow that you are mentioning, and I like this approach better, because it's giving the developer a new tool to use :)

Reply:
So sorry about the delay!!

I agree with @eskombro, I would rather the second approach you suggest 🙂

Sorry again for the delay! Hope you still there @sanders41! 😇

Reply:
No worries @curquiza I'm still here 🙂. I included the update to the contributing file @eskombro and I discussed in my last push. Or do you mean you would like some changes to this update?